h/migrations/versions/3bcd62dd7260_add_featurecohort_table.py

```diff
@@ -0,0 +1,38 @@
+"""add cohort table
+
+Revision ID: 3bcd62dd7260
+Revises: dfa82518915a
+Create Date: 2016-05-10 17:01:02.704596
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '3bcd62dd7260'
+down_revision = 'dfa82518915a'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.create_table('featurecohort',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('name', sa.Unicode(length=100), nullable=False),
```

Comment:
This isn't consistent with the model. The model defines the field as `UnicodeText`, not `Unicode`, and it doesn't specify a length.